src/linkInfo.ts

```diff
@@ -13,11 +13,12 @@ export class LinkInfo {
     public toLink(): string {
         var startIndex = this.position.start.line + 1;
         var endIndex = this.position.end.line + 1;
-        var prefix = `https://${this.siteName}/${this.userName}/${this.repoName}/blob/${this.branchName}/${this.filePath}`;
+        var siteUrlSource = (this.siteName.split("@")[1] === "bitbucket.org") ? 'src' : 'blob';
```

Comment:
basically for bitbucket repositories siteName returns myUserName@siteDomain, and for github repositories it only returns siteDomain, either way, i did this way to work on both.. :)